### PR TITLE
Feature/45630 add license activation prompt

### PR DESCRIPTION
**Pull request by trishasalas**
https://central.tri.be/issues/45630

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | There must be a single space after the first parenthesis | [Checker.php:499](https://github.com/moderntribe/tribe-common/blob/d96b0e42a2f7938245a056dc6668154f2af16958/src/Tribe/PUE/Checker.php#L499)
:x: | There must be a single space before the last parenthesis | [Checker.php:499](https://github.com/moderntribe/tribe-common/blob/d96b0e42a2f7938245a056dc6668154f2af16958/src/Tribe/PUE/Checker.php#L499)
:x: | There must be a space between if and the first parenthesis '(' | [Checker.php:505](https://github.com/moderntribe/tribe-common/blob/d96b0e42a2f7938245a056dc6668154f2af16958/src/Tribe/PUE/Checker.php#L505)

> via jenkins/codesniffer-phpcs

File: src/Tribe/PUE/Checker.php [hunk 1]
```diff
@@ -488,45 +488,25 @@ private function get_api_message( $info ) {
 		}
 
 		/**
-		 * Echo JSON formatted errors
+		 * Displays an error notice if a premium plugin is activated but no license is entered
+		 *
+		 * @since 4.3
```
Comment (borkweb) [inline src/Tribe/PUE/Checker.php]: This method predates 4.3, right?

File: src/Tribe/PUE/Checker.php [hunk 2]
```diff
@@ -488,45 +488,25 @@ private function get_api_message( $info ) {
...
-			if ( version_compare( $pluginInfo->version, $this->get_installed_version(), '>' ) ) {
-				if ( empty( $pluginInfo->api_invalid ) || $pluginInfo->api_invalid != 1 ) {
-					return;
-				}
+			if ( '' != $this->get_plugin_name() ) {
 
-				$msg = $this->get_api_message( $pluginInfo );
-
-				//Dismiss code idea below is obtained from the Gravity Forms Plugin by rocketgenius.com
+				$msg     = $plugin_info['api_invalid_message'];
```
Comment (borkweb) [inline src/Tribe/PUE/Checker.php]: Maybe I'm reading this wrong, but it seems like we are losing the error message provided by `get_api_message()` that was in this method beforehand. Is that intended?

**Comment by trishasalas** [comment]
deleting this branch/PR in order to branch off the branch 🌳

**Comment by barryhughes** [comment]
no point _sticking_ with it

**Comment by trishasalas** [comment]
😂

**Pull request closed without merging**